tests/test_config_generation.py

```diff
@@ -62,3 +64,43 @@ def test_single_geo_disease_set():
     task_configs, _ = generate_task_configs(**validated_args)
     total_tasks_expected = 1
     assert len(task_configs) == total_tasks_expected
+
+
+def test_single_exclusion_generates_number_configs():
+    """Tests that a single disease pair exclusion generates 101 configs."""
+
+    os.environ["task_exclusions"] = "ID:COVID-19"
```

Comment:
Instead of setting these on the environment, let's just define the `args` explicitly, similar to: https://github.com/CDCgov/cfa-config-generator/pull/42/files#diff-542cfd86872c0274c50b531464176db1730d5384554605a0679a7975e2f5e41cR17